### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by GodGun968**

### CODE REVIEW

File: src/main/resources/assets/iris/lang/zh_cn.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
   "iris.shaders.debug.disabled": "调试已禁用。",
   "iris.shaders.debug.failure": "无法启用调试，你的电脑不支持调试功能。",
   "iris.shaders.debug.restart": "没有调试环境，请重启以启用调试。",
+  "iris.load.failure.shader": "光影加载失败！请将错误报告给光影开发人员。",
```
Comment (qznfbnj) [inline src/main/resources/assets/iris/lang/zh_cn.json:9]: 开发者 (Developer)
```suggestion
  "iris.load.failure.shader": "光影包加载失败！请向光影包开发者报告此错误。",
```

File: src/main/resources/assets/iris/lang/zh_cn.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+  "iris.load.failure.generic": "Iris在尝试加载光影时遇到了一个问题；请向Iris开发者报告。",
```
Comment (qznfbnj) [inline src/main/resources/assets/iris/lang/zh_cn.json:10]: Do we need space?
```suggestion
  "iris.load.failure.generic": "Iris 在尝试加载光影时遇到了一个问题；请向 Iris 开发者报告此问题。",
```

File: src/main/resources/assets/iris/lang/zh_cn.json [hunk 3]
```diff
@@ -59,6 +61,8 @@
   "options.iris.shadowDistance.enabled": "允许更改阴影的最远距离。超出此距离的地形和实体不会投射阴影。降低阴影距离可以显著提高性能。",
   "options.iris.shadowDistance.disabled": "你不能改变当前的光影包已经为阴影设置的渲染距离。",
   "options.iris.shadowDistance.sodium_tooltip": "阴影渲染距离控制在阴影通道中可以渲染到多远的地形。更短的距离意味着将渲染更少的地形，从而提高帧率。在明确指定了阴影渲染距离的光影包上无法更改此选项。实际阴影渲染的距离受“渲染距离”设置的限制。",
+  "options.iris.colorSpace": "色彩空间",
+  "options.iris.colorSpace.sodium_tooltip": "将屏幕转换成的色彩空间。在光影之上工作。如果不确定，请使用 SRGB。",
```
Comment (qznfbnj) [inline src/main/resources/assets/iris/lang/zh_cn.json:65]: image
```suggestion
  "options.iris.colorSpace": "颜色空间",
  "options.iris.colorSpace.sodium_tooltip": "将屏幕的颜色空间转换成选中项。在光影之上生效。如果不确定，请使用 SRGB。",
```

**Review by qznfbnj — COMMENTED** [review]
随便看看

**Comment by klkq** [comment]
I sugguest this PR be closed in favor of #2195, for the reason that the author of this PR is not responding to the suggested changes, which are a great improvement and are included in #2195.

**Pull request closed without merging**